### DISCUSSION TITLE
audioタグを使いたい

### DISCUSSION THREAD

**Discussion by agehama** [increments/qiita-discussions#348] [要望・アイデア・不具合]
音声を扱う記事を書いているのですが、その場で再生できるプレイヤーがあると説明するのに便利です。

個人的には記事中でこの書き方が使えると嬉しいです。

```
<audio controls src="..."></audio>
```

音に関する記事は結構たくさんあるので、あったら使うという人は多いのではないでしょうか？
twitter埋め込みで代用している記事はいくつか見ました。

ご検討いただけると幸いです。

**Comment by kyntk** [comment]
@agehama さん
ご意見ありがとうございます。
検討いたします！

**Comment by tsubolabo** [comment]
@kyntk さん
>検討いたします！

期待しております。実現するのはいつ頃になるのでしょうか？
投稿可能フォーマットとしては、mp3以外にm4aも加えていただけると嬉しいです。

**Comment by 6e5d** [comment]
ファイルアップロード機能は現在画像のみ対応しているようですが、音声ファイルにも対応していただけると助かります。

**Reply by tanaka-asami415** [reply]
@6e5d 
ご意見ありがとうございます。
現時点では画像以外のファイルのアップロードを予定していないのですが
ご意見を多くいただくようであれば、検討したいと思います🙇
